docs/content/docs/concepts/append-only-table.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Append Only Table"
+weight: 7
+type: docs
+aliases:
+- /concepts/append-only-table.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Append Only Table
+
+By specifying `'write-mode' = 'append-only'` when creating the table, user creates an append-only table.
+
+You can only insert a complete record into the table. No delete or update is supported and you cannot define primary keys.
+This type of table is suitable for use cases that do not require updates (such as log data synchronization).
+
+## Bucketing
+
+You also need to define bucket number for Append-only table, see [Bucket]({{< ref "docs/concepts/basic-concepts#bucket" >}}).
```

Comment:
```suggestion
You can also define bucket number for Append-only table, see [Bucket]({{< ref "docs/concepts/basic-concepts#bucket" >}}).
```

Users don't "need to" define bucket number. They can always use the default bucket number, which is 1.